### PR TITLE
Update comments about missing exception in http client

### PR DESCRIPTION
**Pull request by trwalke**
Addressing this: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2046#discussion_r526348786

### CODE REVIEW

**Comment by trwalke** [comment]
@henrik-me thoughts on this?

File: src/client/Microsoft.Identity.Client/Platforms/netdesktop/Http/NetDesktopHttpClientFactory.cs [hunk 1]
```diff
@@ -10,6 +10,9 @@ namespace Microsoft.Identity.Client.Platforms.net45.Http
 {
     internal class NetDesktopHttpClientFactory : IMsalHttpClientFactory
     {
+        //Prior to the changes needed in order to make MSAL's httpClients thread safe (https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2046/files),
+        //the httpClient had the possibility to throw an exception stating "Properties can only be modified before sending the first request".
+        //MSAL's httpClient will no longer throw this exception after 4.19.0 (https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/releases/tag/4.19.0)
```
Comment (jennyf19) [inline src/client/Microsoft.Identity.Client/Platforms/netdesktop/Http/NetDesktopHttpClientFactory.cs:15]: what about an aka.ms link w/the info?

Comment (henrik-me) [reply src/client/Microsoft.Identity.Client/Platforms/netdesktop/Http/NetDesktopHttpClientFactory.cs:15]: and how will a customer find this information? I suggest the information is available where we talk about bringing your own httpclient. and should probably be information available in the reference doc where a customer can specify the httpclientfactory as a remark (with the aka.ms link)

---
In reply to: [526570158](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2211#discussion_r526570158) [](ancestors = 526570158)

**Comment by henrik-me** [comment]
I think this is good, however I think the same information should be applied to the public api in the remarks section where we allow specifying a custom httpclientfactory

**Comment by trwalke** [comment]
@henrik-me added comments to api remark

**Review by henrik-me — APPROVED** [review]
:shipit:

**Pull request merged**